.infra/index.ts

```diff
@@ -522,3 +524,48 @@ if (vpcNativeProvider) {
     { provider: vpcNativeProvider.provider },
   );
 }
+
+if (!isAdhocEnv) {
+  new ClickHouseSync(
+    'clickhouse-sync',
+    {
+      isAdhocEnv: isAdhocEnv,
+      namespace: namespace,
+      props: {
+        path: './config.yml',
+        keys: {
+          ...config.requireObject<{ keys: ClickHouseSyncConfig }>(
+            'clickhouseSync',
+          ).keys,
+          'database.hostname': envVars.typeormHost as string,
+          'database.port': '5432',
+          'database.password': config.require('debeziumDbPass'),
+          'database.user': config.require('debeziumDbUser'),
+          'database.server.name': name,
+          'database.dbname': name,
+        },
+        vars: {
+          ...config.requireObject<{ vars: Record<string, string> }>(
+            'clickhouseSync',
+          ).vars,
+        },
+      },
+      image: {
+        repository: 'altinity/clickhouse-sink-connector',
+        tag: '2.0.2-lt',
```

Comment:
```suggestion
        digest:
          'sha256:1c0db9877331a0aaceb2e0f2838db3a5156a3cfbfdb83a92235b6287b576c5d0',
```
We can also use the digest if we want to be sure that the tag isn't force updated

Reply:
What do you think @vpol? You used a forked version (with no changes?) in your PR #1835

Reply:
I use custom build pipeline to build our own images from git repo. so IF and WHEN I need to deploy patched version, I can do it without pain. I do it for other projects as well. we can use our gcr image that I've built and tag.

Reply:
I changed it to use our gcr image. Seeing as we might need to do some patching